pkg/benchmark/benchmark.go

```diff
@@ -130,8 +130,9 @@ func New(cfg *Config) (*Benchmark, error) {
 	}
 
 	for i := range b.workers {
+		var err error
 		w := &worker{
-			id:       uuid.Must(uuid.NewV4()).String(),
+			id:       uuid.Must(uuid.NewV4(), err).String(),
```

Comment:
`uuid.Must` receives a `UUID, error` tuple, which `uuid.NewV4()` returns. Here you'll always inject a `nil` valued error.

Does this actually compile at all for you?

Reply:
... or did the `uuid` library change its upstream signature?

Reply:
signature hasn't changed: https://godoc.org/github.com/satori/go.uuid#NewV4

Additionally you can see the example on the project's README: https://github.com/satori/go.uuid#example

Reply:
I think the problem is that you downgraded the library to v1.2.0, which has a different signature: https://github.com/satori/go.uuid/blob/v1.2.0/generator.go#L64

Reply:
Fixed it by referring to pseudo-version in go.mod file.